Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -3135,6 +3135,31 @@ def visit_Node(self, node):
         return node
 
 
+class FudgeCppTemps(EnvTransform, SkipDeclarations):
```

Comment:
I had to look up the verb "fudge" and still don't think I understand why it's used here. Doesn't "coerce" fit better?
```suggestion
class CoerceCppTemps(EnvTransform, SkipDeclarations):
```